src/js/components/MaskedInput/MaskedInput.js

```diff
@@ -164,6 +164,7 @@ const MaskedInput = forwardRef(
       dropHeight,
       dropProps,
       focus: focusProp,
+      focusIndicator = true,
```

Comment:
I think we need to add `focusIndicator` to MaskedInput/doc.js (and files generated from).